packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -277,7 +277,9 @@ export async function dataToCkEditor(data, type) {
 
 export async function dataToWordCount(data, type) {
   try {
-    const markdown = dataToMarkdown(data, type) || ""
+    const html = await dataToHTML(data, type);
+    const htmlWithoutFootnotes = trimFootnotesFromHTML(html);
+    const markdown = dataToMarkdown(htmlWithoutFootnotes, type) || ""
```

Comment:
Do we need to change `type` to `html` (regardless of what it was originally) if we're now first running it through `dataToHTML` and then trimming it to get rid of the footnotes?